### PR TITLE
Fixed and added some BaseSell prices.

### PR DESCRIPTION
**Pull request by ghost**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

Fixed and added some items NPC prices. Based on Challoux's prices from Port Jeuno. They're neutral (compared to Caiphimonride's prices from Tavnazia Safehold where no fame is involved).

### CODE REVIEW

File: sql/item_basic.sql [hunk 1]
```diff
@@ -14661,7 +14661,7 @@ INSERT INTO `item_basic` VALUES (17436,0,'platinum_rod_+1','platinum_rod_+1',1,2
 INSERT INTO `item_basic` VALUES (17437,0,'curse_wand','curse_wand',1,63572,0,1,0);
 INSERT INTO `item_basic` VALUES (17438,0,'titans_cudgel','titans_cudgel',1,63552,0,1,0);
 INSERT INTO `item_basic` VALUES (17439,0,'leviathans_rod','leviathans_rod',1,63552,0,1,0);
-INSERT INTO `item_basic` VALUES (17440,0,'kraken_club','kraken_club',1,34816,11,0,12850);
+INSERT INTO `item_basic` VALUES (17440,0,'kraken_club','kraken_club',1,34816,11,0,10250);
```
Comment (ibm2431) [inline sql/item_basic.sql:14664]: I am thoroughly impressed.

File: sql/item_basic.sql [hunk 2]
```diff
@@ -19499,15 +19499,15 @@ INSERT INTO `item_basic` VALUES (26178,0,'dimensional_ring_(mea)','dim._ring_(me
 INSERT INTO `item_basic` VALUES (26179,0,'varar_ring','varar_ring',1,2084,25,0,0);
 INSERT INTO `item_basic` VALUES (26180,0,'varar_ring_+1','varar_ring_+1',1,2080,25,0,0);
 INSERT INTO `item_basic` VALUES (26181,0,'chirich_ring','chirich_ring',1,2084,25,0,0);
-INSERT INTO `item_basic` VALUES (26182,0,'chirich_ring_+1','chirich_ring_+1',1,2080,25,0,0);
+INSERT INTO `item_basic` VALUES (26182,0,'chirich_ring_+1','chirich_ring_+1',1,2080,25,0,10587);
 INSERT INTO `item_basic` VALUES (26183,0,'stikini_ring','stikini_ring',1,2084,25,0,0);
-INSERT INTO `item_basic` VALUES (26184,0,'stikini_ring_+1','stikini_ring_+1',1,2080,25,0,0);
+INSERT INTO `item_basic` VALUES (26184,0,'stikini_ring_+1','stikini_ring_+1',1,2080,25,0,12512);
```
Comment (ibm2431) [inline sql/item_basic.sql:19504]: Even my mythic BLM never got one of these!

File: sql/item_basic.sql [hunk 3]
```diff
@@ -19499,15 +19499,15 @@ INSERT INTO `item_basic` VALUES (26178,0,'dimensional_ring_(mea)','dim._ring_(me
...
 INSERT INTO `item_basic` VALUES (26185,0,'niqmaddu_ring','niqmaddu_ring',1,63568,0,1,0);
 INSERT INTO `item_basic` VALUES (26186,0,'ilabrat_ring','ilabrat_ring',1,63568,0,1,0);
 INSERT INTO `item_basic` VALUES (26187,0,'dingir_ring','dingir_ring',1,63568,0,1,0);
 INSERT INTO `item_basic` VALUES (26188,0,'kishar_ring','kishar_ring',1,63568,0,1,0);
 INSERT INTO `item_basic` VALUES (26189,0,'moonbeam_ring','moonbeam_ring',1,2084,25,0,0);
-INSERT INTO `item_basic` VALUES (26190,0,'moonlight_ring','moonlight_ring',1,2080,25,0,0);
+INSERT INTO `item_basic` VALUES (26190,0,'moonlight_ring','moonlight_ring',1,2080,25,0,24062);
```
Comment (ibm2431) [inline sql/item_basic.sql:19510]: Now I see that the purpose of this PR was to show off. 😉

**Review by ibm2431 — APPROVED** [review]
Yep, looks like SQL, Mr. Moneybags! 👍 

Also, you're staff, so insta-merge with this checkmark!

File: sql/item_basic.sql (hunk 2 above)
Comment (kaincenteno) [reply sql/item_basic.sql:19504]: Nyu is OP

**Pull request merged**